client.js

```diff
@@ -159,6 +160,11 @@ bot.on('message', function (message) {
                 message.author.send('Sorry, I didn\'t recognized that command.  Please check your syntax and try again. ' + helpMsg);
                 break;
         }
+        // message the general channel that commands are not recognized in this channel if command detected:
+        if (!message.author.bot && message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL){
+            bot.channels.cache.get(COMMAND_CHANNEL).send(`Hi <@${message.author.id}>, I received your command but deleted it from the \`general\` channel to keep things tidy.  In the future, please use this channel instead or send me a direct message.  Thanks!`)
+            message.delete();
+        }
     }
     else {
```

Comment:
I would expect this else block to activate on normal chat in #general now?

Reply:
This is the same as your comment above I believe? If so, fixed.